### PR TITLE
Added info to note box on 700mb restriction on etcd hosts

### PR DESCRIPTION
**Pull request by bfallonf**
From: https://bugzilla.redhat.com/show_bug.cgi?id=1367035

@gaurav-nelson @ahardin-rh @adellape Peer review plz?

### CODE REVIEW

File: admin_guide/backup_restore.adoc [hunk 1]
```diff
@@ -312,6 +312,11 @@ host:
 ====
 Skip this step if version is lower than `etcd-2.3.7-4` or if etcd database size
 is smaller than 700mb.
+
+If the etcd backup is larger than 700 MB,
+xref:../adming_guide/pruning_resources.html[prune the resource], or clear
```
Comment (ahardin-rh) [inline admin_guide/backup_restore.adoc]: s/adming/admin
s/.html/.adoc  (also remember to include the # and ID (can't be empty)

File: admin_guide/backup_restore.adoc [hunk 2]
```diff
@@ -312,6 +312,11 @@ host:
 ====
 Skip this step if version is lower than `etcd-2.3.7-4` or if etcd database size
 is smaller than 700mb.
```
Comment (ahardin-rh) [inline admin_guide/backup_restore.adoc]: 700mb should be 700 MB

**Comment by ahardin-rh** [comment]
A few comments. Otherwise, LGTM!

**Comment by bfallonf** [comment]
Thanks @ahardin-rh I rushed that link bit... Problems fixed.

I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/backup_restore.adoc#admin-guide-backup-and-restore[Backup and Restore]
|Added to the note with information on host backups over 700 MB.
%

**Pull request merged**